test/distributed/fsdp/test_fsdp_summon_full_params.py

```diff
@@ -308,6 +309,29 @@ def test_summon_full_params_equivalence(self, rank0_only, offload_to_cpu):
 
         self.assertEqual(fsdp_params, params_to_compare)
 
+    @skip_if_lt_x_gpu(2)
+    def test_summon_from_non_fsdp(self):
+        class FSDPContainer(nn.Module):
+            def __init__(self, fsdp_1, fsdp_2):
+                super().__init__()
+                self.fsdp_1 = fsdp_1
```

Comment:
Minor: Should we include `self.fsdp_3` and in both `model_fsdp` and `model_no_fsdp`, we initialize it as `DeterministicModel(wrap_fsdp=False)`?
Knowing the implementation, this test suffices, but to test a more general non-FSDP root module, the root may have non-FSDP children.

Reply:
I see, do you mean add a self.fsdp_3 and `self.fsdp_3 = DeterministicModel(wrap=False)`?

Reply:
Yup.